e2e/fixtures/fdb_cluster.go

```diff
@@ -906,9 +955,11 @@ func (fdbCluster *FdbCluster) UpgradeCluster(version string, waitForReconciliati
 	_ = fdbCluster.GetCluster()
 
 	log.Printf(
-		"Upgrading cluster from version %s to version %s",
+		"Upgrading cluster from version %s to version %s and logServerPerPod %d clusterSpec %d",
```

Comment:
```suggestion
		"Upgrading cluster from version %s to version %s",
```

This was ok during debugging, but we don't want to have these values printed out the whole time.